zipline/assets/futures.py

```diff
@@ -180,3 +180,84 @@ def offset(self, time_delta):
 
         """
         return self.as_of(self.as_of_date + Timedelta(time_delta))
+
+
+# http://www.cmegroup.com/product-codes-listing/month-codes.html
+CME_CODE_TO_MONTH = dict(zip('FGHJKMNQUVXZ', range(1, 13)))
+MONTH_TO_CME_CODE = dict(zip(range(1, 13), 'FGHJKMNQUVXZ'))
+
+
+def cme_code_to_month(code):
+    """
+    Convert a CME month code to a month index.
+
+    The month codes are as follows:
+
+    'F' -> 1  (January)
```

Comment:
Suggest changing the doc string here to read `1 -> 'F' (January)`,  etc., so that it illustrates what the method does.